src/Analyzer.Utilities.UnitTests/JsonLineNumberResolverTests.cs

```diff
@@ -265,6 +266,76 @@ public void ResolveLineNumber_ReturnsCorrectLineNumber(string path, object[] pat
             Assert.AreEqual(expectedLineNumber, resolvedLineNumber);
         }
 
+        [DataTestMethod]
+        [DataRow("resources[0].kind", 10, DisplayName = "")]
+        [DataRow("resources[0].location", 12, DisplayName = "")]
+        [DataRow("resources[0].properties.siteConfig.ftpsState", 15, DisplayName = "")]
+        public void ResolveLineNumber_ContextWithOffset_ReturnsCorrectLineNumber(string path, int lineNumber)
```

Comment:
Could you add a test case to validate that when the template context doesn't specify an offset then 0 will be used?

Reply:
done!